src/ring_theory/unique_factorization_domain.lean

```diff
@@ -526,6 +526,15 @@ begin
   rwa [← normalize_normalized_factor p p_mem, normalize_eq_normalize_iff, dvd_dvd_iff_associated]
 end
 
+lemma normalized_factors_coprime (a : α) : ∀ (p q ∈ normalized_factors a), p ∣ q → p = q :=
```

Comment:
I am not totally convinced by the name, there is no `is_coprime` in the statement.

Reply:
How about:
```suggestion
lemma normalized_factors_eq_of_dvd (a : α) : ∀ (p q ∈ normalized_factors a), p ∣ q → p = q :=
```